qingstor/sdk/client/upload_client.py

```diff
@@ -0,0 +1,70 @@
+import os
+import logging
+
+from file_chunk import FileChunk
+from qingstor.sdk.service.qingstor import QingStor
+from constant import HTTP_CREATED,HTTP_OK,HTTP_BAD_REQUEST
+from error import PartTooSmallError,MaxPartsExceededError,InvalidObjectNameError 
+
+
+class UploadClient:
+    '''file processing,including open file,move file pointer and read file
+
+    Parameter:
+        bucket: bucket refers to bucket object users creating in the QingCloud
+        part_size(int): the part size users want to partition of the file in byte
+        
+    Attributes:
+        file_descriptor(object): the source uploading file object
+        file_name: the key(usually using file name) of the uploading file
+        bucket: bucket refers to bucket object users creating in the QingCloud
+        part_size(int): the part size users want to partition of the file in byte
+        
+    '''
+
+    def __init__(self,bucket,part_size):
+    	if(part_size<4*1024*1024):
+            raise PartTooSmallError()
+        else:
+            self.bucket=bucket
+            self.part_size=part_size
+            self.logger=logging.getLogger("qingstor-sdk")
+        
+    def upload_large_file(self,file_name,file_descriptor):
+        # Initiate multipart upload, create an upload id.
+        output=self.bucket.initiate_multipart_upload(file_name) 
```

Comment:
We should check file size here, if file size is samller than part_size, just do `put_obejct` .